app/src/main/java/org/schabi/newpipe/database/subscription/SubscriptionDAO.kt

```diff
@@ -22,10 +22,42 @@ abstract class SubscriptionDAO : BasicDAO<SubscriptionEntity> {
 
     @Query("""
         SELECT * FROM subscriptions
+
         WHERE name LIKE '%' || :filter || '%'
+
+        ORDER BY name COLLATE NOCASE ASC
+        """)
+    abstract fun getSubscriptionsFiltered(filter: String): Flowable<List<SubscriptionEntity>>
+
+    @Query("""
+        SELECT * FROM subscriptions s
```

Comment:
Is it possible to omit `AS`? If so, I didn't know that.

Reply:
Yes, keep in mind that Room won't let the app compile if any SQL statement is not correct. But it indeed is optional, either as a [column](https://www.sqlite.org/syntax/result-column.html) or a [table alias](https://www.sqlite.org/syntax/table-or-subquery.html).

If I remember correctly, the convention (at least in SQLite) is to use the `AS` keyword when aliasing a column and omit it when aliasing a table. 

###### Not so sure about that though.